examples/model_service_template/gluon_base_service.py

```diff
@@ -122,7 +122,8 @@ def inference(self, data):
         list of NDArray
             Inference output.
         """
-        output = self.net(data)
+        model_input = data.as_in_context(self.ctx)
```

Comment:
Were you facing an issue similar to this ? : https://github.com/apache/incubator-mxnet/issues/13332

Reply:
Yes. It seems to be a similar issue.